_posts/2019-02-12-funding-eslint-future.md

```diff
@@ -0,0 +1,66 @@
+
+Today, we are happy to announce the [ESLint Collective](https://opencollective.com/eslint) on Open Collective. Open Collective is a secure way for people to sustain and support open source projects. The funds taken in and paid out are publicly available on each collective's page, so there is 100% transparency around the money a collective uses. Open Collective has already helped other JavaScript ecosystem projects such as [Webpack](https://opencollective.com/webpack) and [Babel](https://opencollective.com/babel) to fund their ongoing development, and the ESLint team is happy to join them on what has proven to be a great option for open source sustainability.
+
+Some of the benefits of working with Open Collective include:
+
+* **Full Transparency.** Everyone gets to see where money is coming from and where it's going.
+* **Individual and Corporate Sponsors.** Open Collective makes it easy for both individuals and companies to sponsor open source projects.
+* **Potential Tax Benefits.** Because funds are paid to the Open Source Collective, a 501(c)(6) organization in the U.S., there may be tax benefits for some donors (please check with your accountant).
+* **Automatic Invoicing.** For corporate sponsors, Open Collective automatically generates and sends invoices for tracking purposes.
+* **Open Participation.** Anyone can request reimbursement for funds spent helping the ESLint project and ESLint can pay out to anyone.
+
+## How will the money be spent?
+
+The ESLint team has a lot of ideas about how to proceed going forward. Depending on the amount we raise, here are some of the things we're planning:
+
+* **Paying team members for maintenance and development.** All ESLint team members are still contributing in their spare time. Having the option to work part-time or full-time on ESLint, and be compensated for it, would allow for faster and more consistent development.
+* **Rewarding contributions from the community.** We'd like to reward everyone who contributes to ESLint, whether it's a one-time contribution or ongoing contributions. We are still figuring out the details, but rewarding people with everything from stickers and t-shirts to cash payouts for significant contributions are things we are thinking about right now.
+* **Improve documentation.** ESLint's documentation hasn't had a significant overhaul since the project was first released. We feel there are a lot of ways to improve the documentation, including potentially contracting an information architect or technical writer to improve our documentation.
+* **Better communication and support.* By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
```

Comment:
"meetups, companies, and conferences": Two of these are events, one (companies) is a group of entities/locations. This feels like a small parallelism issue, but I'm not sure how best to resolve it (or, for that matter, if it even needs to be resolved).

Also, formatting typo:

```suggestion
* **Better communication and support.** By paying team members part-time or full-time, that also opens up the possibility of sending team members to meetups, companies, and conferences to interact with the community and better understand the problems developers are facing.
```

Reply:
Yeah, I don't think it's important in this context. All of them are places we couldn't afford to go previously.